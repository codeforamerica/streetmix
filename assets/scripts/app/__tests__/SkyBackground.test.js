/* eslint-env jest */
import React from 'react'
import { SkyBackground } from '../SkyBackground'
import { shallow } from 'enzyme'

describe('SkyBackground', () => {
  it('renders', () => {
    const wrapper = shallow(<SkyBackground environment="foo" scrollPos={0} height={100} />)
    expect(wrapper).toMatchSnapshot()
  })
})
