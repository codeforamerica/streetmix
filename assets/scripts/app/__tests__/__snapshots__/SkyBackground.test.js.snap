// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SkyBackground renders 1`] = `
<section
  className="street-section-sky"
  style={
    Object {
      "height": "100px",
    }
  }
>
  <div
    className="sky-background"
  >
    <div
      className="sky-background-previous"
      key="prev-foo"
      style={
        Object {
          "background": "#a9ccdb",
        }
      }
    />
    <div
      className="sky-background-current"
      key="current-foo"
      style={
        Object {
          "background": "#a9ccdb",
        }
      }
    />
  </div>
  <SkyBackgroundObjects
    objects={Array []}
  />
  <div
    className="rear-clouds"
    style={
      Object {
        "WebkitTransform": "translateX(0px)",
        "opacity": null,
        "transform": "translateX(0px)",
      }
    }
  />
  <div
    className="front-clouds"
    style={
      Object {
        "WebkitTransform": "translateX(0px)",
        "opacity": null,
        "transform": "translateX(0px)",
      }
    }
  />
  <div
    className="sky-foreground"
    style={
      Object {
        "opacity": 0,
      }
    }
  />
</section>
`;
